app/shared/ui-comment/src/androidMain/kotlin/ui/comment/Turnstile.android.kt

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2024 OpenAni and contributors.
+ *
+ * 此源代码的使用受 GNU AFFERO GENERAL PUBLIC LICENSE version 3 许可证的约束, 可以在以下链接找到该许可证.
+ * Use of this source code is governed by the GNU AGPLv3 license, which can be found at the following link.
+ *
+ * https://github.com/open-ani/ani/blob/main/LICENSE
+ */
+
+package me.him188.ani.app.ui.comment
+
+import android.annotation.SuppressLint
+import android.graphics.Color
+import android.os.Bundle
+import android.webkit.WebResourceRequest
+import android.webkit.WebResourceResponse
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import android.widget.FrameLayout.LayoutParams
+import android.widget.Toast
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.Stable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Constraints
+import androidx.compose.ui.viewinterop.AndroidView
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.flow.receiveAsFlow
+import me.him188.ani.app.platform.LocalContext
+import me.him188.ani.app.ui.foundation.ProvideFoundationCompositionLocalsForPreview
+import me.him188.ani.app.ui.foundation.widgets.LocalToaster
+import me.him188.ani.app.ui.foundation.widgets.Toaster
+import java.io.InputStream
+
+@Stable
+class AndroidTurnstileState(
+    override val url: String,
+) : TurnstileState {
+    private val webViewState = mutableStateOf<WebView?>(null)
+    var webView: WebView?
+        get() = webViewState.value
+        set(value) {
+            if (webViewState.value == value) return
+            webViewState.value = value
+            value?.applySettings()
+            value?.restoreOrLoadPage()
+        }
+
+    var isDarkTheme: Boolean = false
```

Comment:
This property violates `@Stable` becuase when it changes, it does not notify Compose runtime.

Make it `private` or a state.